src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -174,6 +174,10 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('list', 'acr_task_list')
         g.command('delete', 'acr_task_delete')
         g.command('update', 'acr_task_update')
+        g.command('credential add', 'acr_task_credential_add')
+        g.command('credential remove', 'acr_task_credential_remove')
```

Comment:
I see in acr commands, we use `delete` instead of `remove`

Reply:
I do see in some cases when working with a collection we use `add` entity to or `remove` entity from the collection, such as our own `az acr network-rule remove` command that follows the pattern from everyone else.

Also see https://english.stackexchange.com/a/52509 as an informal reference.